front/public/locales/en/rollingstock.json

```
@@ -69,7 +69,8 @@
     "rollingStockDuplicateName": "Rolling stock of the same name already exists.",
     "rollingStockNotAdded": "Rolling stock not added",
     "rollingStockNotUpdated": "Rolling stock not updated",
-    "rollingStockNotDeleted": "Rolling stock not deleted"
+    "rollingStockNotDeleted": "Rolling stock not deleted",
+    "invalidEffortCurves": "Speed and effort values must be strictly positive, and speed values must be unique (without duplicates)."
```

See comment in utils

```suggestion
    "invalidEffortCurves": "Speed values must be unique (without duplicates)."
```